12_glob_lab/globlab.txt

```diff
@@ -5,25 +5,90 @@ https://www.tocode.co.il/bundles/unix1/lessons/globlab
 
   1. All files whose name starts with a lowercase letter
   Answer: 
+  #amir@ubuntu:~/wildcards$ find ./ -name "[[:lower:]]*.*"
+  #./Doc/test.c
+  #./lib/zlib.c
+  #./img/hero.png
+  #./img/player.png
+  #./src/player.c
+  #./src/main.c
+  #./include/player.h
+
 
   2. All files whose name starts with an uppercase letter
   Answer:
 
+  # amir@ubuntu:~/wildcards$ find ./ -name "[[:upper:]]*.*"
+  # ./Doc/README.txt
+
   3. All files whose name ends in an "a", "e", "i", "o" or "u"
   Answer:
+  # amir@ubuntu:~/wildcards$ find ./ -name "*[aeiou].*"
+  #./img/hero.png
 
   4. All files whose name DOES NOT end in an "a", "e", "i", "o" or "u"
   Answer:
+  # amir@ubuntu:~/wildcards$ find ! -name  "*[aeiou].*"
+  #.
+  #./Doc
+  #./Doc/test.c
+  #./Doc/README.txt
+  #./lib
+  #./lib/zlib.c
+  #./img
+  #./img/player.png
+  #./src
+  #./src/player.c
+  #./src/main.c
+  #./include
+  #./include/player.h
+
 
   5. All files with extension ".c" that are in a folder named "src"
   Answer:
+  #amir@ubuntu:~/wildcards$ find ./src  -name "*.[c]"
```

Comment:
The `[c]` is not required here, since c is the only option. Better to just use `*.c`